scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ConnectionIdGenerator.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ *
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ *
+ * Contributors:
+ *     Achim Kraus (Bosch Software Innovations GmbH) - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.californium.scandium.dtls;
+
+import org.eclipse.californium.elements.util.DatagramReader;
+
+/**
+ * Connection id generator.
```

Comment:
Should we add more documentation ?
```
A connection ID generator. 
Responsible for generating ID which identifies Scandium connection in store.

By default, DTLS defined that IP address and port of the peer are used to identify the DTLS Connection.
The DTLS connection ID draft defines a way to identify connection using Connection ID and so supports environments where IP address/port changes .
{@link https://tools.ietf.org/html/draft-ietf-tls-dtls-connection-id-04}

This class can be used to activate support of this draft and configure if the current peer want foreign peers use connection ID when sending message.
```

Reply:
The "04" has some typos too much :-). 
See my issues there. I adjusted the javadoc with the 03.